nss-day-cohort-46/rare-client-groovy-goblins
# Cr categories delete

# Description

Only logged-in user can delete categories.

## Type of change


# Testing Instructions

`git fetch --all`
`git checkout cr-categories-delete`
`npm start`

For server-side code, see backend pull request.

If logged in as staff/admin, after logging in, click on `Category Management`.
Click any category to delete.
Page will rerender to show category deleted.

If logged in as non- staff/admin, after logging in, click on `Category Management`.
Delete option will not be available.


# Checklist:
